### ISSUE TITLE
Does Glimpse support websocket packets capturing?

### ISSUE THREAD

**Issue by crazyyi** [Glimpse/Home#102]
I tried to view request/response sent via websockets using Glimpse. But I can't see any. My test environment doesn't even go forward as it used to be when I init Glimpse in my code. It will stop when it tries to send the first websocket packet. Does Glimpse support capturing websocket events? Glimpse is a great project. It has a friendly UI and is easy to use. But it is not very useful for me if it can't view websocket events.

**Comment by nikmd23** [comment]
Hello @crazyyi,

Glimpse does not currently provide diagnostic insights into web sockets, but we'd love to work with you more to understand how and why you're using websockets, and the problems you experience while doing so. Mind filling in some details covering how you're using websockets, what modules/libraries/frameworks you leverage and what you'd find helpful from Glimpse?

You also mention that your "test environment doesn't go forward". Call you tell us a bit more about what you mean by that? It sounds like you might have found a bug, but we need more info.

Thanks for writing in!

**Comment by crazyyi** [comment]
Sure. I am just running a test environment using the `ws` and `socket.io` library. But my main goal is to use Glimpse to analyze the data on the wire when I run my browser game in websocket. Otherwise I have to rely on console logs on the browser which doesn't give me a very good experience.

**Comment by avanderhoorn** [comment]
So given a magic wand, you are wanting to see the connection that has been established and the messages going back and forward? What in these messages are you looking to find information on?

**Comment by crazyyi** [comment]
@avanderhoorn yes to see the established connection and debugging and profiling.

**Comment by avanderhoorn** [comment]
Technically that should be doable. Whats your socket setup like? i.e. is it like a chat client where you have multiple channels going and routing messages between groups/user or is it more for "real time" updates/notifications to the use.

**Comment by crazyyi** [comment]
@avanderhoorn It's real time MMO server-client browser game.

**Comment by avanderhoorn** [comment]
Makes sense. What are the payloads like in this case, are they just plain old json objects or are they more complex?

**Comment by crazyyi** [comment]
message format can be json object or binary stream. The payloads depend on the number of connected clients. If there are 100 connected clients and each of them receives 10 messages per second then the server will be sending 1000+ messages out per second. The client also sends messages to the server very often.
